package.json

```diff
@@ -7,9 +7,9 @@
     "test": "jest",
     "semantic-release": "semantic-release",
     "postinstall": "rm -f node_modules/web3/index.d.ts node_modules/web3/types.d.ts",
-    "build:ts": "ttsc -b --watch .",
+    "build:ts": "ttsc -b .",
```

Comment:
We should have a separate script that uses `--watch` for development. Maybe `build:ts-dev`?

Reply:
Reverted npm script `build:ts` to its original and updated `build:prod`